vignettes/TADAModule1.Rmd

```diff
@@ -213,7 +224,7 @@ to dates, datetimes, and numerics based on a standard algorithm.
 Enter ?TADAdataRetrieval into the console to review more example queries
 and additional information.
 
-```{r}
+```{r TADAdataRetrieval}
 TADAProfile <- TADAdataRetrieval(organization = c("REDLAKE_WQX", "SFNOES_WQX", "PUEBLO_POJOAQUE", "FONDULAC_WQX", "PUEBLOOFTESUQUE", "CNENVSER"), startDate = "2018-01-01")
```

Comment:
This is a larger dataset example, which is really nice for trouble shooting and scoping what a user might experience. However, it takes forever to run through the vignette with all of the steps and devtools::check() takes 30+ minutes. Might suggest cutting it down to 2 years of data, or less.

Reply:
Good idea - it does take forever. I'll put an end date of January 1, 2020, and see if that helps.